data/human/news.txt

```diff
@@ -1413,7 +1450,7 @@ news "tourist"
 			"Can you tell me again which planet this is? No, I'm not lost, I just don't know where I am."
 			"We did a ride on an ancient spacecraft the other day. It was called something like Soyuz or Shuttle Dragon. I don't remember the name exactly, but it was horrible; my back still hurts from the bumpy ride."
 			"I almost missed my transport last week; someone was wrong on the hypernet and I was in a long, heated discussion to correct it."
-			"Hey! Where's platform 5? I need to board there in a few minutes!"
+			"Hey! Where's Platform ${digit}? I need to board there in a few minutes!"
```

Comment:
I don't understand exactly how the substitutions work, but there's a `"platform number"` phrase in this file that should probably be used here.

Reply:
Thanks for the suggestion. It has been implemented  with the latest commit.

If you're interested in the new substitution/interpolation syntax, it is documented [here](https://github.com/endless-sky/endless-sky/wiki/WritingConversations#text)